ddtrace/propagation/_database_monitoring.py

```diff
@@ -122,3 +123,25 @@ def _get_dbm_comment(self, db_span):
             # replace leading whitespace with trailing whitespace
             return sql_comment.strip() + " "
         return ""
+
+
+def handle_dbm_injection(int_config, span, args, kwargs):
+    dbm_propagator = getattr(int_config, "_dbm_propagator", None)
+    if dbm_propagator:
+        args, kwargs = dbm_propagator.inject(span, args, kwargs)
+
+    return span, args, kwargs
+
+
+def handle_dbm_injection_asyncpg(int_config, method, span, args, kwargs):
+    # bind_execute_many uses prepared statements which we want to avoid injection for
+    if method.__name__ != "bind_execute_many":
+        return handle_dbm_injection(int_config, span, args, kwargs)
+    return span, args, kwargs
+
+
+if dbm_config.propagation_mode in ["full", "service"]:
+    dbm_integrations = ["aiomysql", "dbapi"]
+    for integration in dbm_integrations:
+        core.on(f"{integration}.execute", handle_dbm_injection)
```

Comment:
optional: I'd prefer for each to be listed independently, so I can do a code search for registration, otherwise this bit is hard to find

Reply:
Okay, done